k9mail-library/src/test/java/com/fsck/k9/mail/store/imap/ImapConnectionTest.java

```diff
@@ -901,6 +901,39 @@ public void sendContinuation() throws Exception {
         server.verifyInteractionCompleted();
     }
 
+    @Test
+    public void executeSingleCommand_on_openConnection_shouldReturnResult() throws Exception {
+        MockImapServer server = new MockImapServer();
+        simpleOpenDialog(server, "");
+        server.expect("4 CREATE Folder");
+        server.output("4 OK Folder created");
+
+        ImapConnection imapConnection = startServerAndCreateImapConnection(server);
+        List<ImapResponse> result = imapConnection.executeSimpleCommand("CREATE Folder");
+        assertEquals(result.size(), 1);
+
+        server.verifyConnectionStillOpen();
+        server.verifyInteractionCompleted();
+        server.shutdown();
```

Comment:
`verifyInteractionCompleted()` calls `shutdown()` internally. So this line can be removed.